src/main/java/frc/robot/commands/ShootPasserCommand.java

```diff
@@ -12,12 +12,24 @@ public class ShootPasserCommand extends Command {
     private Passer passer = Passer.getInstance();
     private Timer timer = new Timer();
     private double secondsToScore = 0;
+    private double shootDelay = 0.25;
+
+    public ShootPasserCommand(double secondsToScore, double shootDelay) {
+        this.addRequirements(scoring);
+        this.secondsToScore = secondsToScore;
+        this.shootDelay += shootDelay;
+    }
 
     public ShootPasserCommand(double secondsToScore) {
         this.addRequirements(scoring);
         this.secondsToScore = secondsToScore;
     }
 
+    public ShootPasserCommand(double shootDelay, boolean doShootDelay) {
+        this.addRequirements(scoring);
+        this.shootDelay += shootDelay;
+    }
+
```

Comment:
I don't think there is a need for this constructor since the programmer can choose not to delay shooting by using the second constructor and delay shooting with the first constructor.